features/integration-github-bitbucket-gitlab.md

```diff
@@ -14,3 +14,60 @@ You can then either create a new organization directly synchronized with a GitHu
 
 **Update web hooks** notifying Packagist of new code pushes will be installed in the background, and **credentials to access the package** will be **configured automatically**.
+
+## Integrations
+
+Private Packagist integrates with the following systems:
+
+#### GitHub
+* OAuth: Users authenticate on Private Packagist with their GitHub accounts
+* Synchronization:
+    * Keeps teams, their members, and access permissions in sync with your GitHub organisation
+    * The default repository permissions on Github is used to grant members access to your repositories
+* Code Credentials: GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### GitHub Enterprise
+* OAuth: Users authenticate on Private Packagist with their GitHub accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/github-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps team members and access permissions in sync with your GitHub Enterprise organisations
+* Code Credentials: GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### Bitbucket.org / Cloud
+* OAuth: Users authenticate on Private Packagist with their Bitbucket.org accounts
+* Synchronization:
+    * Keeps groups, their members, and access permissions in sync with your Bitbucket team
+* Code Credentials: Bitbucket API Key or Bitbucket App Password
+* Webhooks: Code changes and releases
+
+#### Bitbucket Server / Stash
+* OAuth: Users authenticate on Private Packagist with their Bitbucket accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/bitbucket-server-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps users and access permissions in sync with your Bitbucket projects
+    * Individual collaborators aren't supported
+* Code Credentials: personal access token which are available since Bitbucket Server 5.5 or username and password for older versions
+* Webhooks: Code changes and releases
+
+#### GitLab
+* OAuth: Users authenticate on Private Packagist with their GitLab accounts
+* Synchronization:
+    * Keeps access permissions in sync with your Gitlab groups
+    * Individual collaborators aren't supported
+* Code Credentials: GitLab API token
+* Webhooks: Code changes and releases
+
+#### GitLab (Self-hosted)
+* OAuth: Users authenticate on Private Packagist with their GitLab accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/gitlab-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps access permissions in sync with your Gitlab groups
+    * Individual collaborators aren't yet supported
+* Code Credentials: GitLab API Token
+* Webhooks: Code changes and releases
+
+#### AWS CodeCommit 
+* Synchronization:
+    * Possible after manually setting up webhooks
```

Comment:
A synchronization in Private Packagist refers to an entire organization this is not possible with AWS CodeCommit.